platform/android/src/style/sources/custom_vector_source.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
```

Comment:
This is a left-over after refactoring right?

Reply:
Yeah - this is definitely a mistake.